docs/u-boot.md

```diff
@@ -0,0 +1,32 @@
+# U-Boot Support
+MilvusVisor has support for U-Boot environment.
+We tested this on QEMU.
+
+## How to build
+To enable U-Boot support, you need to enable `u-boot` feature flag.
+
+```shell
+rustup component add rust-src
+cd path/to/repo-root/src
+make custom_all FEATURES=u_boot
+```
+
+## How to execute on QEMU
+After you build MilvusVisor with U-Boot support, you can run it by following steps
+
+1. Install `qemu-system-aarch64`
+2. Build your U-Boot for QEMU
```

Comment:
This document assume that the built binary is at `path/to/u-boot-root/u-boot.bin`? If so, could you add this assumption here?